.github/workflows/preflight-checks.yaml

```diff
@@ -86,16 +84,6 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
       - uses: actions/checkout@v2
-      - name: Build node modules
-        run: |
-          cd web
-          npm ci --prefer-offline --no-audit
-          npm run-script build
-        shell: bash
-      - uses: actions/upload-artifact@v2
```

Comment:
Why is this removed? https://github.com/vmware-tanzu/octant/blob/af6ee1bcee8ab61f87f9949982e1fe308650c8ed/.github/workflows/preflight-checks.yaml#L140-L143 This step will download nothing if it is a release

Reply:
Oh It's my bad. I didn't realize that this is being used across jobs. I see the same problem with nightly.yml then which has goreleaser using it. I'll revert it.